lib/src/fields/text_field.dart

```diff
@@ -0,0 +1,1386 @@
+import 'dart:ui' as ui show BoxHeightStyle, BoxWidthStyle;
+
+import 'package:flutter/foundation.dart' show defaultTargetPlatform, kIsWeb;
+import 'package:flutter/gestures.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
+
+import 'package:macos_ui/macos_ui.dart';
+import 'package:macos_ui/src/library.dart';
+
+export 'package:flutter/services.dart'
```

Comment:
Why are we exporting these libraries here?

Reply:
We are only exporting the following classes:
```dart
export 'package:flutter/services.dart'
    show
        TextInputType,
        TextInputAction,
        TextCapitalization,
        SmartQuotesType,
        SmartDashesType;
```
These are necessary in order to have the best dev experience. For example, if I want to change the text field type to email on mobile, I would need to import `TextInputType` from `flutter/services` because it's already exported here.

Reply:
Okay